using-packages.md

```diff
@@ -18,6 +18,9 @@ platform SDKs (like
 
 If you are looking to develop a new package, please see [developing packages](/developing-packages/).
 
+If you are looking to add assets, images, or fonts to your application, please
```

Comment:
maybe hint as to why this is related to Using Packages ?

e.g. "Some package provide assets such as fonts and images. Learn more...."

Reply:
Would just "If you are looking to add assets, images, or fonts, whether stored in files or packages, please..." work?

Reply:
sure!